docs/maintainers/deploying.md

```diff
@@ -5,30 +5,23 @@ title: Deployment Guide
 # Deploying Forem
 
 Anyone with the ability to merge PRs on GitHub can deploy the application.
-Remember that this is a shared responsibility, so everyone should deploy code
-occasionally.
+Whenever a PR is merged the code is deployed. When deploying complex code, be
+sure that other team members are around to help if something goes wrong.
```

Comment:
```suggestion
**Whenever a PR is merged the code is deployed. When deploying complex code, be
sure that other team members are around to help if something goes wrong.**
```

Not a blocker, just an idea so that we make sure that no one misses this point 😂